src/js/components/SearchBar/InputBackdrop.js

```diff
@@ -0,0 +1,24 @@
+/* @flow */
+import styled from "styled-components"
+
+import type {Styled} from "../../types/styled"
+
+const Input: Styled<> = styled.div`
```

Comment:
This is fun. I realized I could have a much easier time laying out the search bar if I made a containing "div" look like the search input. Then the actual input where the user types is completely transparent. I could then use flexbox to layout the buttons instead of padding and absolute positions.

This component is the background.

<img width="524" alt="image" src="https://user-images.githubusercontent.com/3460638/91353407-17e6b280-e7a0-11ea-9913-f52edc29303b.png">